app/views/access_permissions/partials/_public_members_dropdown.html.erb

```diff
@@ -5,7 +5,14 @@
       <div class="title">
         <%= t('.title', team: team.name) %>
       </div>
-      <% team.users.order(full_name: :asc).where.not(id: project.manually_assigned_users.select(:id)).each do |user| %>
+      <%
+        users_excluded = if defined? project
```

Comment:
this should be generalized to 'assignable', instead of switching between project and protocol

Reply:
variable should be called `excluded_user_ids`